lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -960,8 +961,13 @@ def _create_deployment_parameters(
             arm_parameters.admin_key_data = get_public_key_data(
                 self.runbook.admin_private_key_file
             )
-        else:
+        elif self.runbook.admin_password:
             arm_parameters.admin_password = self.runbook.admin_password
+        else:
+            (
+                arm_parameters.admin_key_data,
```

Comment:
The key should be the same to every environment, so it's easier to use.

Reply:
How about `_initialize`?

Reply:
ok. in get_or_generate_key_pairs, if the key exists (means has been created) then reuse it

Reply:
updated